test-framework/ui/src/main/java/org/keycloak/testframework/ui/util/PageUtils.java

```diff
@@ -0,0 +1,26 @@
+package org.keycloak.testframework.ui.util;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class PageUtils {
+
+    public static String getPageTitle(WebDriver driver) {
+        try {
+            return driver.findElement(By.id("kc-page-title")).getText();
+        } catch (NoSuchElementException e) {
+            return null;
+        }
+    }
```

Comment:
Probably not needed, but if it is then just add to AbstractPage

Reply:
ok